addons/stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -84,3 +84,9 @@ class StockPicking(models.Model):
         check_company=True,
         states={'done': [('readonly', True)], 'cancel': [('readonly', True)]},
         help='Batch associated to this transfer', copy=False)
+
+    def _should_show_transfers(self):
+        if len(self.batch_id) == 1:
```

Comment:
why ?

Reply:
if all the pickings you're processing are linked to the same batch it means you're processing the batch


you coud process two pickings from two different batchs from the lsit view